processing/create_mosaic.py

```diff
@@ -21,7 +21,7 @@
 import matplotlib.pyplot as plt
 from skimage.exposure import equalize_adapthist, rescale_intensity
 from skimage.transform import resize
-
+sys.path.append(os.path.join(sys.path[-1], "scripts"))
```

Comment:
i don't think this is going to work-- you want to append to sct dir, not sys.path[-1], which could vary based on people's bashrc)

Reply:
That why I thought it was an ugly fix, but the `from spinalcordtoolbox.utils import __sct_dir__` seem to work